js/search/tuning.js

```
@@ -0,0 +1,31 @@
+var fs = require('fs');
+
+var utils = require('./utils');
+
+function findAllWords() {
+	var data = fs.readFileSync('./index_default.json');
+	var raw = JSON.parse(data).documentStore.store;
+	var words = [];
+	for (var key in raw) {
+		words = words.concat(raw[key]);
+	}
+	return words;
+}
+
+function findStopWords() {
+	var words = findAllWords();
+	if (words.length < 100) {
+		console.log('ERROR');
```

do want to just log the error do you want to throw the error? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error